test/torchaudio_unittest/functional/functional_impl.py

```diff
@@ -227,6 +227,24 @@ def test_mask_along_axis_iid(self, mask_param, mask_value, axis):
         assert mask_specgrams.size() == specgrams.size()
         assert (num_masked_columns < mask_param).sum() == num_masked_columns.numel()
 
+    def test_resample_no_warning(self):
+        sample_rate = 44100
+        waveform = get_whitenoise(sample_rate=sample_rate)
```

Comment:
nit: since we do not care about the resulting tensor, we can use something much smaller than a Tensor with 44100 elements, while keeping the value of `sample_rate = 44100`.